src/main/java/pl/fracz/mcr/comment/Comment.java

```diff
@@ -0,0 +1,41 @@
+package pl.fracz.mcr.comment;
+
+import pl.fracz.mcr.source.Line;
+
+import java.io.File;
+
+public class Comment extends AbstractComment {
+    private String text;
+
+    private File file;
+
+    public Comment(AbstractComment.Type type, Line line) {
+        super(type, line.get());
+    }
+
+    public void setText(String text) {
```

Comment:
Lepiej moze to w ogole osobne klasy porobic TextComment VoiceComment itd bo taki sprawdzanie to jak dojdzie 100 typow komentarzy bd lipa